project/[[ python_package_distribution_name ]]/__init__.py.jinja

```diff
@@ -7,4 +7,4 @@
 
 from typing import List
 
-__all__: List[str] = []  # noqa: WPS410 (the only __variable__ we use)
+__all__: List[str] = []  # noqa: WPS410 (the only __variable__ we use)
```

Comment:
If we are going to leave the __init__.py then it should have a new line here.

Reply:
Updated. use following comment to test
'copier git@github.com:kefeimo/copier-poetry-volttron-agent.git output/ -r b692591'